pennylane/operation.py

```diff
@@ -1129,7 +1131,8 @@ def expand(self):
             # original operation has no trainable parameters
             tape.trainable_params = {}
 
-        if self.inverse:
+        # the inverse attribute can be defined by subclasses
```

Comment:
Worth extending this comment: why is it important that
> the inverse attribute can be defined by subclasses

?

Reply:
Yes I could look into it, I just left this note to say why we are disabling the check, but why the code was written this way I don't know.

Reply:
In this case, even though Operator accesses the inverse attribute, it does not assign it anywhere in it's definition. The Operation subclass e.g. does define it though.